aries-site/src/pages/components/card.mdx

```diff
@@ -69,34 +147,132 @@ A card by nature has limited real-estate, so it should present summarized inform
 
-This type of card is more commonly used within marketing contexts and should be used sparingly within HPE products and services. Secondary purposes might include an action to navigate to a screen with greater detail on the offering.
+#### Anatomy
+
+DIAGRAM TO BE ADDED
+
+1. **Card identifier**: This region is intended to provide the user with context on what the card is about.
+   - a. **Title** (required): For Call-to-action cards, this should be a unique, enticing headline to capture the user's attention.
+   - b. **Pretitle** (optional): Additional context for what this call-to-action is about (e.g., "Virtual Event").
+   - c. **Subtitle** (optional): Any concise, additional supportive text to help establish the card's identity and context.
+1. **Actions** (required): Card-level actions.
+   - A CTA, primary, or solid-filled button labelled with the main call-to-action is required. At times, additional supporting actions/links may be appropriate.
+1. **Description** (optional): A short paragraph that elaborates on the card's purpose.
+1. **Image/Media** (optional): An eye-catching, [brand-approved image](https://brandcentral.hpe.com/brand-central/content/distinctive-brand-assets/flexible-ingredients#p1-photography-style-1) that grabs the user's attention. Use of this area should be reserved for highly promotional cards like events and feature announcements.
+1. **Background** (optional): A background color or [brand-approved texture/gradient](https://brandcentral.hpe.com/brand-central/content/distinctive-brand-assets/flexible-ingredients#c3-color-and-texture-style-3).
+   - When using a texture or gradient, be mindful to maintain appropriate color contrast between the card's text and the background.
+
+#### Presentation
+
+- Call-to-action cards may be presented in a group with other call-to-action cards but also may be presented on their own.
+- When presented in a group, their layout/treatment can be independent from the layout/treatment of the surrounding call-to-action cards.
+
+- Cards that are highly promotional should use elements that create higher visual emphasis (colorful brand-approved images, background colors/textures, etc.).
+  - Use cases (ordered from highest visual emphasis to lowest) include event promotion, new feature release, trial promotion.
+- Cards that are less promotional should still have enticing titles but do not necessarily need as strong of visual emphasis. The [white brand-approved textures](https://brandcentral.hpe.com/brand-central/content/distinctive-brand-assets/flexible-ingredients#c3-color-and-texture-style-3) are appropriate if a subtle visual emphasis is desired.
+  - A use case is a software upgrade announcement.
+
+#### Examples
+
+These are example layouts for various use cases. Call-to-action cards that are highly promotional, such as event promotions, incorporate stronger visual elements while those that are less promotional, like software updates, are more subtle.
```

Comment:
```suggestion
These are example layouts for various use cases.
```
I don't think we need to include the info about the stronger visual elements vs more subtle visual elements because we mention this in the section above as well as in the examples, just feels slightly repetitive

Reply:
That's fair. I think I had this portion before I had the "Presentation" portion. Removing.